app/main.py

```diff
@@ -16,4 +16,11 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"immutable": [lucky_number,
+                                  pi,
+                                  one_is_a_prime_number,
+                                  name,
+                                  profile_info],
+                    "mutable": [my_favourite_films,
+                                marks,
+                                collection_of_coins]}
```

Comment:
Consider putting each key with its corresponding value on a new line for better readability.